File: server/thirdeye/src/main/java/com/aeye/thirdeye/entity/Project.java
package com.aeye.thirdeye.entity;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import javax.validation.constraints.Size;
import java.util.ArrayList;
import java.util.List;

@Entity
@Table(name = "Project")
@Getter
@Setter
@NoArgsConstructor//(access = AccessLevel.PROTECTED)
public class Project {
    @Id
    @GeneratedValue
    private Long id;

    @Size(max = 50)
    private String provider;

    @Size(max = 50)
    private String title;

    private int goal;

    @Size(max = 1024)
    private String description;

    @OneToMany(mappedBy = "project")
    private List<Image> imageList = new ArrayList<>();
}

File: mobile2/NextLabel/app/src/main/java/com/aeye/nextlabel/model/dto/Project.kt
package com.aeye.nextlabel.model.dto

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Project(
    val accepted: Int,
    val description: String,
    val goal: Int,
    val id: Int,
    val provider: String,
    val title: String
) : Parcelable